internal/access/resource_access_workflow.go

```diff
@@ -144,6 +149,26 @@ func (r *AccessWorkflowResource) Create(ctx context.Context, req resource.Create
 		createReq.ActivationExpiry = durationpb.New(activationExpiry)
 	}
 
+	var defaultDuration time.Duration
```

Comment:
Can you update this to just set default duration to access duration by default. Helps readability.

```suggestion
	//set default duration to access duration by default
	defaultDuration := accessDuration
	if !data.DefaultDuration.IsNull() {
		//set default duration if it is set
		defaultDuration = time.Second * time.Duration(data.DefaultDuration.ValueInt64())

		if defaultDuration >= accessDuration {
			resp.Diagnostics.AddError(
				"Invalid Default Duration",
				"The default duration must be less than the maximum access duration. "+
					"Please adjust the Default Duration to be less than Access Duration.\n\n"+
					"Default Duration: "+defaultDuration.String()+", Access Duration: "+accessDuration.String(),
			)
			return
		}

	}
```